docs/UserGuide.md

```diff
@@ -86,57 +83,64 @@ Format for adding a new asset: `add --id <asset id> --location <location>`
 * Any number of digits are allowed.
 * ‘+’ is optional and must be the first character.
 
-`--tag`
-* Tag associated with contact.
-* Contact can be created first without tags, then tags can be added later.
-* Case sensitive, i.e. NUS != nus.
-* Must have no spaces.
-
-`--id`
-* An ID associated with the asset.
-* Not necessarily unique.
+`--address`
+* Address of the contact.
 
-`--location`
-* The location associated with the asset.
+`--asset`
+* Asset(s) associated with contact.
+* Contact can be created first without assets, then assets can be added later using the Edit command.
+* Case sensitive, i.e. NUS != nus.
```

Comment:
Nit: should we use computing notation `!=` in a user guide?

Reply:
Replaced with ≠